src/renderers/dom/shared/__tests__/DOMPropertyOperations-test.js

```diff
@@ -174,10 +174,12 @@ describe('DOMPropertyOperations', function() {
 
   describe('setValueForProperty', function() {
     var stubNode;
+    var stubInstance;
 
     beforeEach(function() {
       stubNode = document.createElement('div');
-      ReactDOMComponentTree.precacheNode({}, stubNode);
+      stubInstance = {_debugID: 1};
+      ReactDOMComponentTree.precacheNode(stubInstance, stubNode);
```

Comment:
The stub instances used to be plain `{}` which doesn’t have `debugID` so it turned out to be `undefined`. I err on the side of specifying them where necessary rather than omitting them for convenience, so that we can have stronger warnings in `ReactDebugTool`.